### PR TITLE
sd-bus: Downgrade D-Bus signal log messages to trace level

### PR DESCRIPTION
**Pull request by daandemeyer**
These are extremely noisy and not very useful so let's downgrade them to trace level.

### CODE REVIEW

**Review by bluca — APPROVED** [review]

File: src/basic/unit-file.c [hunk 1]
```diff
@@ -567,8 +567,6 @@ int unit_file_build_name_map(
                                         if (!dst)
                                                 return log_oom();
                                 }
-
-                                log_debug("%s: normal unit file: %s", __func__, dst);
```
Comment (bluca) [inline src/basic/unit-file.c:571]: Make it trace instead? For when things are desperate, more logs are better than fewer

**Comment by poettering** [comment]
uh, sorry, this stuff is really my bread and butter. When is this noisy? It's *so* helpful to see this info. I am not convinced at all we should drop this.

**Comment by DaanDeMeyer** [comment]
> uh, sorry, this stuff is really my bread and butter. When is this noisy? It's _so_ helpful to see this info. I am not convinced at all we should drop this.

I understand that you care about method calls and replies (and so do I), but you care about signals?

**Comment by poettering** [comment]
> I understand that you care about method calls and replies (and so do I), but you care about signals?

yes i do, because they are usually triggerd by method calls/replies

**Review by yuwata — COMMENTED** [review]
I'd like to keep all of them. These are only shown on reload or reexec, and sometimes really useful to debug what happen during that operations.

**Pull request closed without merging**